test/driver.cpp

```diff
@@ -148,6 +161,7 @@ void processCmdLine(std::vector<std::string_view> arguments)
       std::cout << msg << '\n'; //TODO: print in appropriate stream based in cmd-line
```

Comment:
Message needs to be printed to std::cout or fileOut depending on tester configuration. Assuming fileOut is set to
```suggestion
      (outputFilename.empty() ? cout : fileOut )<< msg << '\n';
```